Xamarin.Android.sln

```diff
@@ -348,6 +352,18 @@ Global
 		{B8F799C5-D7CE-4E09-9CE6-BAA4173E7EC8}.XAIntegrationDebug|AnyCPU.Build.0 = Debug|Any CPU
 		{B8F799C5-D7CE-4E09-9CE6-BAA4173E7EC8}.XAIntegrationRelease|AnyCPU.ActiveCfg = Debug|Any CPU
 		{B8F799C5-D7CE-4E09-9CE6-BAA4173E7EC8}.XAIntegrationRelease|AnyCPU.Build.0 = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.Debug|AnyCPU.ActiveCfg = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.Debug|AnyCPU.Build.0 = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.Release|AnyCPU.ActiveCfg = Release|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.Release|AnyCPU.Build.0 = Release|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.XAIntegrationDebug|AnyCPU.ActiveCfg = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.XAIntegrationDebug|AnyCPU.Build.0 = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.XAIntegrationRelease|AnyCPU.ActiveCfg = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.XAIntegrationRelease|AnyCPU.Build.0 = Debug|Any CPU
+		{5EB9E888-E357-417E-9F39-DDEC195CE47F}.XAIntegrationDebug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
This is similarly odd; your IDE is adding `XAIntegrationDebug` *twice*: once for `AnyCPU` and once for `Any CPU` (note space). (Ditto `XAIntegrationRelease`.)

Sounds like we should file a bug against that IDE... :-/